### PR TITLE
Update responsible employee

### PR DESCRIPTION
**Pull request by RossSihovsk**
Added ability to update Responsible Employee
Added functional with History of changes

### CODE REVIEW

**Review by Max877866 — APPROVED** [review]

**Review by Natalia1910 — APPROVED** [review]

File: service/src/main/java/greencity/service/ubs/EventServiceImpl.java [hunk 1]
```diff
@@ -35,4 +36,37 @@ public void save(String eventName, String eventAuthor, Order order) {
...
+     * This method return correct status for changes with Responsible employee.
+     *
+     * @param positionId    ID changed position.
+     * @param existedBefore If True - We update Info, otherwise we assign
+     * @author Rostyslav Sikhovskiy.
+     */
+    @Override
+    public String changesWithResponsibleEmployee(Long positionId, Boolean existedBefore) {
+        if (existedBefore.equals(Boolean.TRUE)) {
+            if (positionId == 2) {
```
Comment (thesvider) [inline service/src/main/java/greencity/service/ubs/EventServiceImpl.java]: can you define these relations in map instead?

File: service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java [hunk 2]
```diff
@@ -2226,8 +2226,17 @@ public void updateOrderAdminPageInfo(UpdateOrderPageAdminDto updateOrderPageDto,
...
+            if (nonNull(updateOrderPageDto.getUpdateResponsibleEmployeeDto())) {
+                updateOrderPageDto.getUpdateResponsibleEmployeeDto().stream()
+                    .forEach(dto -> ordersAdminsPageService.responsibleEmployee(List.of(orderId),
+                        dto.getEmployeeId().toString(),
+                        dto.getPositionId(),
+                        currentUser));
+            }
         } catch (UnexistingOrderException | PaymentNotFoundException | UserNotFoundException | UBSuserNotFoundException
-            | NotFoundOrderAddressException | ReceivingStationNotFoundException | OrderNotFoundException e) {
+            | NotFoundOrderAddressException | ReceivingStationNotFoundException | OrderNotFoundException
```
Comment (thesvider) [inline service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java]: too many exceptions. I'd rather catch "Exception" than defining all of that.

Comment (RossSihovsk) [reply service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java]: Fixed

File: service/src/main/java/greencity/service/ubs/EventServiceImpl.java (hunk 1 above)
Comment (RossSihovsk) [reply service/src/main/java/greencity/service/ubs/EventServiceImpl.java]: Fixed

**Review by KuzbytMaksym — APPROVED** [review]

**Pull request merged**